<ui version="4.0" >
 <class>properties</class>
 <widget class="QWidget" name="properties" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>178</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <widget class="QLabel" name="appNameLabel" >
       <property name="palette" >
        <palette>
         <active>
          <colorrole role="WindowText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Button" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Light" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Midlight" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Dark" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>127</red>
             <green>127</green>
             <blue>127</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Mid" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>170</red>
             <green>170</green>
             <blue>170</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="BrightText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Shadow" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="AlternateBase" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ToolTipBase" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>220</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ToolTipText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Button" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Light" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Midlight" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Dark" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>127</red>
             <green>127</green>
             <blue>127</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Mid" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>170</red>
             <green>170</green>
             <blue>170</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="BrightText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Shadow" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="AlternateBase" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ToolTipBase" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>220</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ToolTipText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>127</red>
             <green>127</green>
             <blue>127</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Button" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Light" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Midlight" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Dark" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>127</red>
             <green>127</green>
             <blue>127</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Mid" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>170</red>
             <green>170</green>
             <blue>170</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>127</red>
             <green>127</green>
             <blue>127</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="BrightText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>127</red>
             <green>127</green>
             <blue>127</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Shadow" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="AlternateBase" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ToolTipBase" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>220</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ToolTipText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="autoFillBackground" >
        <bool>true</bool>
       </property>
       <property name="frameShape" >
        <enum>QFrame::Panel</enum>
       </property>
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" >
       <item>
        <widget class="QLabel" name="versionLabel" >
         <property name="text" >
          <string>Version number:</string>
         </property>
         <property name="buddy" >
          <cstring>kcfg_appVersion</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KLineEdit" name="kcfg_appVersion" >
         <property name="toolTip" >
          <string>Project's version number</string>
         </property>
         <property name="whatsThis" >
          <string>Set your project version number. A first project should start with version 0.1.</string>
         </property>
         <property name="inputMask" >
          <string>0.0; </string>
         </property>
         <property name="text" >
          <string>0.1</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" >
       <item>
        <widget class="QLabel" name="installLabel" >
         <property name="text" >
          <string>Installation directory:</string>
         </property>
         <property name="buddy" >
          <cstring>kcfg_url</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KUrlRequester" name="kcfg_url" >
         <property name="toolTip" >
          <string>The directory where you will build your project</string>
         </property>
         <property name="whatsThis" >
          <string>Choose the directory where you will build your project. Your home /src is a good default location.</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4" >
       <item>
        <widget class="QLabel" name="authorLabel" >
         <property name="text" >
          <string>Author name:</string>
         </property>
         <property name="buddy" >
          <cstring>kcfg_name</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KLineEdit" name="kcfg_name" >
         <property name="toolTip" >
          <string>Your first name and name</string>
         </property>
         <property name="whatsThis" >
          <string>This will set the copyright to this name</string>
         </property>
         <property name="inputMask" >
          <string/>
         </property>
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5" >
       <item>
        <widget class="QLabel" name="emailLabel" >
         <property name="text" >
          <string>Author email:</string>
         </property>
         <property name="buddy" >
          <cstring>kcfg_email</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KLineEdit" name="kcfg_email" >
         <property name="toolTip" >
          <string>Your email address</string>
         </property>
         <property name="whatsThis" >
          <string>This email address will be next to your name in the copyright credit of the project files.</string>
         </property>
         <property name="inputMask" >
          <string/>
         </property>
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
